server/html_data/index.html

```diff
@@ -88,7 +88,7 @@ <h3 id="statusTitle" class="panel-title">
         top_block.appendTo(parent_block);
     }
     function handleResponse(response, rootId) {
-        var totalFullyOperatedPlatforms, totalUnreachablePlatforms = 0;
+        var totalFullyOperatedPlatforms = 0, totalUnreachablePlatforms = 0;
```

Comment:
Might be better to have them on individual lines. The initial problem would have been more easily avoided then.

Reply:
Ok. Will make as part of other changes